docs/administration/netbox-shell.md

```diff
@@ -143,6 +145,8 @@ To return the inverse of a filtered queryset, use `exclude()` instead of `filter
 346
 ```
 
+Queries can all also be executed from a particular object instead of from the model itself. For instance, to get all circuits that are assigned to one site, it is easier to filter from the site itself, instead of using the "Circuit" model and building the query from there. This is particularly useful for configuration templates and export templates, since it allows to query other database objects that are related to the object that we're rendering the template for. The same methods (all, filter, exclude, get...) can be used in this kind of queries.
```

Comment:
Again, this is redundant to the Django documentation, which provides plenty of context and examples.